src/styles/block/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars__star {
+  height: 12px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
```

Comment:
1) apply background `shorthand`
2) probably you need to use background-position: center;